Dockerfile.acceptance-testing

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.11 as builder
+FROM golang:1.13 as builder
```

Comment:
Thanks for the update. The main `Dockerfile` was already using 1.13, so now tests and binary are using the same :+1: